option optimize_for = LITE_RUNTIME;

message SellChipRequest
{
 repeated uint32 pos = 1;
}
